data/categories.txt

```diff
@@ -1881,6 +1881,7 @@ da:^Jernbaneoverskæring
 fi:^Tasoristeys
 hu:^Útátjáró
 fa:تقاطع هم سطح با راه اهن
+el:^Ισόπεδη σιδ/μική διάβαση
```

Comment:
The synonyms are separated by the vertical bar (```|```) in this file.
The '^' sign marks the synonym that should be visible when showing an object of this category on the map.

Reply:
> σιδ/μική is a shortcut of σιδηροδρομική... a long word
> we may use Ισόπεδη διάβαση τρένου, even with or without τρένου

Please use the full word. You may add all three versions if you think it will help.
The "σιδ/μική" will likely be split into "σιδ" and "μική", even if it looks meaningless. And anyway, a user should use exactly this form of shortening to succeed in their search.

So, what we are trying to do here is to guess what a user would type in order to search for a category of objects.